.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This folder should be removed from the PR. I believe the up-to-date .gitignore includes this folder.